tests/Database/DatabaseEloquentRelationTest.php

```diff
@@ -37,9 +37,40 @@ public function testTouchMethodUpdatesRelatedTimestamps()
 		$relation->touch();
 	}
 
+    /**
+     * Executing parent model's global scopes could result in an infinite loop when the parent model's
```

Comment:
Please format this so there is a short description of no more than 80 chars inc the whitespace on the same line before the comment, and then a long description with line wrapping of 80 chars. Formatted like this.

```php
    /**
     * Short description.
     *
     * Long description.
     */
```